staging/src/k8s.io/client-go/tools/cache/reflector.go

```diff
@@ -49,7 +51,13 @@ type Reflector struct {
 	metrics *reflectorMetrics
 
 	// The type of object we expect to place in the store.
+	// Set if the type is not Unstructured.
 	expectedType reflect.Type
```

Comment:
expectedNativeType

Reply:
And move the name on-top of this, so that the native and the unstructured fields are next to each other.

Reply:
Done